account_banking_sepa_credit_transfer/models/account_payment_order.py

```diff
@@ -85,40 +84,46 @@ def generate_payment_file(self):
             priority = line.priority
             local_instrument = line.local_instrument
             categ_purpose = line.category_purpose
+            line.compute_sepa()
```

Comment:
this is suspicious: computed fields should update themselves automatically if the dependencies are correct